js/gateio.js

```diff
@@ -770,7 +771,7 @@ module.exports = class gateio extends Exchange {
     }
 
     prepareRequest (market) {
-        if (market['contract']) {
+        if (market['derivative']) {
```

Comment:
Why is this changed? `market['contract']` is used everywhere else

Reply:
we've changed `contract` to mean the contractSize is not one, aka you have to be careful when making orders and derivative means futures aka swap or future

Reply:
> we've changed contract to mean the contractSize is not one,

Why is that what it means?

Reply:
@frosty00 i think it's a different meaning right now. Specifically, this:

> We could have derivative: true for all contracts and contract: true only for those where you trade contracts (amount precision in contracts).